clients/sensu-aws-client/check.go

```diff
@@ -145,12 +215,12 @@ func AWSCheck() check.ExtensionCheckResult {
 
 	failedInstances := []string{}
 
-	for _, status := range r.InstanceStatus {
-		log.Println(status.InstanceId)
-		if status.InstanceState.Code == 16 &&
-			(status.SystemStatus.Status != "ok" ||
-				status.InstanceStatus.Status != "ok") {
-			failedInstances = append(failedInstances, status.InstanceId)
+	for _, status := range r.InstanceStatuses {
+		log.Println(*status.InstanceId)
```

Comment:
because in the official `aws/aws-sdk-go` implemetation it's a `*string` and there is no helpers to render the pointer as a value